### ISSUE TITLE
lec 6. Multinomail classification

### ISSUE THREAD

**Issue by sculove** [studye/yeahEE#7]
### multinomail classification 
- 여러개의 binary classfication
- A,B,C는 A와 그 외, B와 그 외, C와 그외

![2017-10-26 9 48 16](https://user-images.githubusercontent.com/1383952/32053836-66794b1c-ba97-11e7-9839-11afaa56f7c4.png)
![2017-10-26 9 48 38](https://user-images.githubusercontent.com/1383952/32053857-71bd5464-ba97-11e7-8e0d-3fe55dc3b5f6.png)
![2017-10-26 9 50 24](https://user-images.githubusercontent.com/1383952/32053944-b3b95b38-ba97-11e7-8a11-4561f567fa25.png)

### softmax => sigmod
각각의 n개의 값을 softmax에 넣으면 전체 합이 1인 값이 나옴.
- 0~1사이값
- 전체의 합이 1 (각각이 확률로 볼수 있다)
- one-hot encoding 방법으로 하나만 고르게 한다.

![2017-10-26 9 50 46](https://user-images.githubusercontent.com/1383952/32053961-c08b73d2-ba97-11e7-85f8-a3e6e98a6e1a.png)

### cross-entropy => cost 함수를 구하는 기본 함수
![2017-10-26 9 52 44](https://user-images.githubusercontent.com/1383952/32054029-0537c47c-ba98-11e7-9ff9-f3e52e3abb4a.png)
![2017-10-26 9 54 47](https://user-images.githubusercontent.com/1383952/32054110-529578c2-ba98-11e7-8588-4d6c081970b9.png)

### cost 함수는  cross-entropy의 합의 평균값
